Add docs warning about reload and debug flags (#949) (#1045) (#1047)
Kludex/uvicorn: docs/settings.md

If you're running using programmatically, using `uvicorn.run(...)`, then use
equivalent keyword arguments, eg. `uvicorn.run("example:app", port=5000, reload=True, access_log=False)`.

> Please note, in this scene, if you need reloading feature (`reload=True`),

Reviewer: ```suggestion
Please note that in this case, if you use `reload=True` or `debug=True`,
```
